sematic/ui/packages/addon-grafana/.eslintrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
Please add the bullet point instructions from the PR description on how to include one of these UI plugins in a dev document somewhere. Maybe in the existing [dev documentation manual](https://github.com/sematic-ai/sematic/blob/main/docs/developers/manual.md), or in a new doc for the UI?

Reply:
Thanks. Doc is added to [docs/developers/manual.md](https://github.com/sematic-ai/sematic/pull/976/files#diff-465dda3f4d175981ff3e4ca70674d1425cb5e14a754629523bb2c236644c0beb) for now.